dist/Cwel/Pattern/card-slider/card-slider.e2e.js

```diff
@@ -0,0 +1,89 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
@NathanRodger @dstuessy Do we commit e2e files to dist?

Reply:
@NathanRodger @Decad I think so cause this way we can run these tests on other projects if need be.